assembl/static2/js/app/components/profile/avatar.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
```

Comment:
Is it normal that the component is called avatar but does not contain any avatar (picture) of a user?